packages/bundler-plugin-core/src/utils/providers/GitHubActions.ts

```diff
@@ -136,30 +137,15 @@ function _getSHA(
     return args.sha;
   }
 
-  const pr = _getPR(inputs);
+  const context = GitHub.context;
 
   let commit = envs?.GITHUB_SHA;
-  if (pr) {
-    const mergeCommitRegex = /^[a-z0-9]{40} [a-z0-9]{40}$/;
-    const mergeCommitMessage = runExternalProgram("git", [
-      "show",
-      "--no-patch",
-      "--format=%P",
-    ]);
-
-    debug(`Merge commit message: ${mergeCommitMessage}`, {
-      enabled: output.debug,
-    });
-
-    if (mergeCommitRegex.exec(mergeCommitMessage)) {
-      const splitMergeCommit = mergeCommitMessage.split(" ");
-
-      debug(`Split commit message: ${splitMergeCommit}`, {
-        enabled: output.debug,
-      });
-
-      commit = splitMergeCommit[1];
-    }
+  if (
+    `${context.eventName}` == "pull_request" ||
+    `${context.eventName}` == "pull_request_target"
```

Comment:
```suggestion
    `${context.eventName}` === "pull_request" ||
    `${context.eventName}` === "pull_request_target"
```

Also do we have to coerce `context.eventName` to be a string?

you can also do an array.includes check to avoid duplicating `${context.eventName}` ===`

```js
["pull_request", "pull_request_target"].includes(context.eventName)
```

ditto with usage below.

Reply:
Darn ... that is a big goof on my end, too much after hours PHP for me ... looks like it's typed properly so no need to coerce. I wonder if they may have ran into type issues [here](https://github.com/codecov/codecov-action/blob/main/src/buildExec.ts#L35-L36), and so they wrapped it in the string to make it easier?